tests/integration/files/conf/master

```diff
@@ -1,24 +1,25 @@
 id: master
-user: ubuntu
 interface: 127.0.0.1
 publish_port: 64505
 ret_port: 64506
 worker_threads: 3
 pidfile: master.pid
+user: ubuntu
```

Comment:
The user should not be set, the test suite takes care of identifying the user running the tests and adding that to the configuration which is generated.